webpack/assets/javascripts/react_app/redux/actions/statistics/index.js

```diff
@@ -0,0 +1,18 @@
+import API from '../../../API';
+import {
+  STATISTICS_DATA_REQUEST,
+  STATISTICS_DATA_SUCCESS,
+  STATISTICS_DATA_FAILURE
+} from '../../consts';
+
+export const getStatisticsData = charts => dispatch => {
+  dispatch({ type: STATISTICS_DATA_REQUEST, payload: charts });
+  charts.forEach(c => {
+    API.get(c.url).then(
+      result => dispatch({ type: STATISTICS_DATA_SUCCESS, payload: result }),
+      (jqXHR, textStatus, error) => dispatch(
+        { type: STATISTICS_DATA_FAILURE, payload: { error, id: c.id } }
+      )
+    );
+  });
+};
```

Comment:
Will the above replace 'webpack/assets/javascripts/react_app/actions/ServerActions.js' in the event that we completely replace flux with redux.